en/docs/learn/jwt-token-generation.md

```diff
@@ -163,3 +163,22 @@ public class ValidateRSASignature {
     }
 }
 ```
+### JWT validation at introspection
+
+To enable the server to identify a JWT token at the user info endpoint and token introspection endpoints add the following config.
+
+```toml
+[oauth]
+enable_jwt_token_validation_during_introspection = true
+```
+
+If identified, validation is performed by treating the token as a JWT access token.
+
+!!! Important 
+    Enabling this feature will validate the token using the available JWT token validator. Ex: In the
+    default pack, this will be Default JWT token validator. However, if the server issue custom JWT
+    tokens, which fails the available JWT token validator, the existing flow can break. For an example,
+    the default JWT token validator mandate to have the jti claim in the token. If a custom token issuer
+    does not include this claim, introspection breaks once this feature is enabled. In such scenarios,
+    a compatible JWT token validator should be deployed in the system prior to enabling this feature.
```

Comment:
```suggestion
!!! Important 
    Enabling this feature will validate the token using the available JWT token validator. For example, in the default pack, this will be the default JWT token validator. However, the existing flow can break if the server issues custom JWT tokens, which invalidates the available JWT token validator. For example, the default JWT token validator mandates having the `jti` claim in the token. If a custom token issuer does not include this claim, the introspection process breaks once this feature is enabled. In such scenarios, a compatible JWT token validator should be deployed in the system before enabling this feature.
```

Reply:
Resolved in [5b14f6f](https://github.com/wso2/docs-is/pull/3895/commits/5b14f6fe0854ea51d3e2360f2d8a9cebf314a8ca)